sound/soc/intel/boards/sof_sdw.c

```diff
@@ -935,7 +936,7 @@ static int create_sdw_dailink(struct device *dev, int *be_index,
 			      playback, capture,
 			      cpus + *cpu_id, cpu_dai_num,
 			      codecs, codec_num,
-			      NULL, &sdw_ops);
+			      NULL, &sdw_ops, true);
```

Comment:
Adding booleans to init_dai_link() makes it even harder to follow. How about just setting nonatomic to true here in a separate call outside init_dai_link (with a comment that notes SDW uses nonatomic stream ops and thus the nonatomic flag needs to be set)?  No need to modify other places where init_dai_link is called.